src/domain/mainwindow.py

```diff
@@ -241,7 +250,7 @@ def update_info_widget(self):
                 str(self.operation_mode.last_classified_animals))
 
     def url_input_dialog(self):
-        """Method to run dialog for insertion of an URL to fetch image from."""
+        """Method to run dialog for insertion of a URL to fetch image from."""
```

Comment:
I think that "an" is correct, instead of "a"

Reply:
I got educated that what matter is the sound of the initial letter. if it sounds like vowel then 'a' applies... Pycharm suggestion :)